### PR TITLE
Update conformance-tests.md

### PR DESCRIPTION
**Pull request by jayunit100**
note on > 1 node for conformance

### CODE REVIEW

File: contributors/devel/sig-architecture/conformance-tests.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ Examples of tests which are not eligible to conformance:
     - If the test is checking for specific conditions or reasons, it is considered
       overly specific and it is recommended to simply look for pass/failure criteria
       where possible, and output the condition/reason for debugging purposes only.
+- a cluster MUST have > 1 node, to be defined as Conformant.
```
Comment (dims) [inline contributors/devel/sig-architecture/conformance-tests.md:87]: Doesn't seem like a good spot for this info @jayunit100

Comment (dims) [reply contributors/devel/sig-architecture/conformance-tests.md:87]: how about the "Running Conformance Tests" section below?

**Comment by nikhita** [comment]
Ping @jayunit100 :)

**Pull request closed without merging**